srcpkgs/linux-asahi/template

```diff
@@ -0,0 +1,349 @@
+# Template file for 'linux-asahi'
+pkgname=linux-asahi
+version=6.12.4p1
+revision=1
+short_desc="Linux kernel and modules for Apple Silicon"
+maintainer="Will Springer <skirmisher@protonmail.com>, dkwo <npiazza@disroot.org>"
+_majver=${version%p*}
+_asahiver=${_majver%.0}
+_asahirel=${version#*p}
+_tag=asahi-${_asahiver}-${_asahirel}
+license="GPL-2.0-only"
+homepage="https://asahilinux.org"
+distfiles="https://github.com/AsahiLinux/linux/archive/${_tag}.tar.gz"
+checksum=6543a6aa3fadb65fa24d5b7d4ae2143054229a94bc309512ce7c436104048e2f
+
+python_version=3
+
+archs="aarch64*"
+
+nodebug=yes  # -dbg package is generated below manually
+nostrip=yes
+noverifyrdeps=yes
+noshlibprovides=yes
+preserve=yes
+
+hostmakedepends="tar xz bc elfutils-devel flex gmp-devel kmod libmpc-devel
+ pkg-config openssl-devel perl uboot-mkimage cpio pahole python3 zstd
+ rust cargo rust-bindgen rust-src libclang"
+
+_kernver=${_majver}-${_tag}_${revision}
```

Comment:
maybe `asahi-${version}_${revision}`? or something

Reply:
This messes up with the sed of CONFIG_LOCAL_VERSION: how about i just drop asahi here then?

Reply:
it could be `${version}_${revision}-asahi` or something else like that, or just drop the asahi

Reply:
I think it works now.